Backoffice/Security/Authorization/Voter/ContentVoter.php

```diff
@@ -72,16 +75,17 @@ protected function voteForOwnedSubject($action, $content, UserInterface $user)
      *
      * @param string                  $action
      * @param ContentInterface|string $subject
-     * @param UserInterface           $user
+     * @param TokenInterface          $token
      *
      * @return bool
      */
-    protected function voteForSomeoneElseSubject($action, $subject, UserInterface $user)
+    protected function voteForSomeoneElseSubject($action, $subject, TokenInterface $token)
     {
         if (!is_object($subject)) {
             return false;
         }
 
+        $user = $token->getUser();
```

Comment:
@alavieille : there's no more the need to create the local var $user, you can directly inject $token->getUser() in the condition, or better, you can inject the $token in `isSubjectInPerimeter`. That way if we refacto the token to store the perimeter, the modification here will be small